sdk/Sdk/Targets/Microsoft.Azure.Functions.Worker.Sdk.props

```diff
@@ -20,6 +20,9 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
         <ProjectCapability Include="AzureFunctions"/>
   </ItemGroup>
 
+  <ItemGroup>
+    <CompilerVisibleProperty Include="FunctionsDisableFuncMetadataSourceGen" />
```

Comment:
Very open to name suggestions - this is just a first thought that I used for prototyping

Reply:
Only note I would have is removing fun in the middle but I'm fine as is: `FunctionsDisableMetadataSourceGen`

Reply:
I left another comment with a naming suggestion: `FunctionsMetadataSourceGen_Enabled`. That comment has an explanation for why.